corehq/motech/dhis2/repeaters.py

```diff
@@ -31,6 +31,10 @@ class Meta(object):
 
     dhis2_config = SchemaProperty(Dhis2Config)
 
+    @classmethod
```

Comment:
I have questions about this. Originally I used `has_config = False` in the parent class, and `has_config = True` here, but when I used it in a Django template False values were truthy! It seemed to become a `BooleanProperty` instance (which I didn't understand) and Django saw an instance of a thing and treated it as True (which I also didn't understand). 

I gave up easily, and just used a method. But thanks for pointing this out. It would be much better to figure out what was happening here, and resolve it. Any idea why/how it became a (jsonschema) Property?

Reply:
Huh, I don't know what's going on, either. I played with it a bit and saw the same behavior. I tried removing `QuickCachedDocumentMixin` from `Repeater` and it didn't make a difference, so I assume it's something about how `Document` handles properties...I don't know enough about Document/couchdbkit/jsonobject to make an educated guess.

@dannyroberts @emord do you know why this happens?

Reply:
Can you try it with `_allow_dynamic_properties = False` ? I'm not 100% that will work or the exact reason why that happens, but it "feels" like it could solve it.

Reply:
Yeah `Document` (a subclass of `JsonObject`) will convert any top-level properties to jsonobject "Properties". On instances, these will behave like whatever the type of the property is. On classes, however, these Property things are just objects.

So yeah that's the behavior, not necessarily saying it's great, but it's something we inherited from couchdbkit when we rewrote this part of it many years ago, and something I imagine our code widely depends on at this point.

That said, a little tip is that you can avoid that behavior by prefixing it with a `_`. So `_has_config = True` or whatever would be treated the way you want I think, except of course that you'd have to call it `blah._has_config`.

Reply:
Thanks for the help everyone! @dannyroberts, that's really useful.